packages/react/src/components/form/form-field-context.ts

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext } from 'react';
+
+export interface FormFieldControlProps {
+    formId?: string;
+    ariaLabel?: string;
+    ariaLabelledby?: string;
+    ariaDescribedby?: string
+    disabled?: boolean;
+    required?: boolean;
+    valid?: boolean;
+}
+
+export const FormFieldContext = createContext<FormFieldControlProps>({});
+
+function applyDefault<T>(fields: T, defaults: T, keys: (keyof T)[]): T {
+    const result = <T>{};
```

Comment:
On ne devrait jamais utiliser les casting avec les brackets. `as` fait la meme chose pis il marche partout.

```suggestion
    const result = {} as T;
```